### ISSUE TITLE
Corner label with button - icon not in the right place

### ISSUE THREAD

**Issue by moshfeu** [fomantic/Fomantic-UI#518]
# Feature Request

## Description
It's working well only on `a` tag while, I think, the most usage for label like this is `button` (like button like in the example). I know that I can use `a` tag as button but it's bad semantic.

## Example

```html
<div class="ui two column grid">
  <div class="column">
    <div class="ui fluid image">
      <button class="ui left corner label">
        <i class="heart icon"></i>
      </button>
      <img src="https://fomantic-ui.com/images/wireframe/image.png">
    </div>
  </div>
  <div class="column">
    <div class="ui fluid image">
      <a class="ui red right corner label">
        <i class="save icon"></i>
      </a>
      <img src="https://fomantic-ui.com/images/wireframe/image.png">
    </div>
  </div>
</div>
```

## Testcase (when possible)
https://jsfiddle.net/moshfeu/h4582kbc/3/

## Screenshot (when possible)
<img width="586" alt="screen shot 2019-02-25 at 22 25 50" src="https://user-images.githubusercontent.com/3723951/53366489-623d6800-394c-11e9-81df-6d591e8f12fc.png">

Based on https://github.com/Semantic-Org/Semantic-UI-React/issues/3462

**Comment by lubber-de** [comment]
Prepared a fix in #520

**Comment by moshfeu** [comment]
Thanks!
